src/NodeJS/Server.php

```diff
@@ -46,6 +46,11 @@
     protected $threshold;
 
     /**
+     * @var array
+     */
+    protected $options;
```

Comment:
please use private visibility for the new property (as it has both a getter and a setter which are public, there is no benefit making this accessible through inheritance, and protected stuff makes it harder to handle backward compatibility)